stories/documentation/structure/page-header/angular/page-header-basic.stories.ts

```diff
@@ -0,0 +1,146 @@
+	<button type="button" luButton="text" luTooltip="Supprimer" luTooltipOnlyForDisplay><lu-icon icon="trashDelete" alt="Supprimer" /></button>
+</ng-container>`
+			: ``;
+		const backActionContainer = backAction
+			? `<ng-container pageHeaderBackAction>
+		<a href="#" luButton="text">
+			<lu-icon icon="arrowLeft" alt="Retour" />
+		</a>
+</ng-container>`
+			: ``;
+		const navigationContainer = navigation
+			? `<ng-container pageHeaderNavigation>
+	<lu-horizontal-navigation >
+		<a *luHorizontalNavigationLink class="horizontalNavigation-list-item-action" href="#" aria-current="page">Page</a>
+		<a *luHorizontalNavigationLink class="horizontalNavigation-list-item-action" href="#">Page</a>
+		<a *luHorizontalNavigationLink class="horizontalNavigation-list-item-action" href="#">Page</a>
+	</lu-horizontal-navigation>
+</ng-container>`
+			: ``;
+		const actionsContainer = actions
+			? `<ng-container pageHeaderActions>
+		<lu-form-field
+			label="Label"
+			hiddenLabel
+		>
+			<lu-text-input
+				hasSearchIcon
+				type="text"
+				placeholder="ex : Mon précieux"
+				[(ngModel)]="example"
+			></lu-text-input>
+		</lu-form-field>
+		<button type="button" luButton>Button</button>
+		<button type="button" luButton="outline">Button</button>
+		<button type="button" luButton="text"><lu-icon icon="menuDots" alt="Voir plus d’options" /></button>
+	</ng-container>`
+			: ``;
+		const breadcrumbsContainer = breadcrumbs
+			? `<ng-container pageHeaderBreadcrumbs>
+		<lu-breadcrumbs>
+			<a *luBreadcrumbsLink class="breadcrumbs-list-item-action" routerLink="/" ariaCurrentWhenActive="page">Page 0</a>
+			<a *luBreadcrumbsLink class="breadcrumbs-list-item-action" ariaCurrentWhenActive="page" href="#2">Page 1</a>
+			<a *luBreadcrumbsLink class="breadcrumbs-list-item-action" aria-current="page">Page 2</a>
+		</lu-breadcrumbs>
+	</ng-container>`
+			: ``;
+		const desc = descriptionPortalContent ? `[description]="description"` : ``;
+		const title = labelPortalContent ? `[label]="label"` : ``;
+		const templateDescription = descriptionPortalContent
+			? `<ng-template #description>
+	<p>
+		Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed ac justo scelerisque, blandit nibh quis, imperdiet justo.
+		Nullam condimentum nulla et neque ultricies bibendum
+		<a href="#" luLink external>Lien</a>.
+	</p>
+</ng-template>`
+			: ``;
+		const templateLabel = labelPortalContent
+			? `<ng-template #label><h1 luTooltip="Tooltip">
+		H1. Page title
+</h1>
+</ng-template>`
+			: ``;
+		return {
+			template: `
+${templateDescription}
+${templateLabel}
+<lu-page-header ${desc} ${title} ${generateInputs(otherArgs, argTypes)}>
+	${breadcrumbsContainer}
+	${backActionContainer}
+	${titleActionsContainer}
+	${actionsContainer}
+	${navigationContainer}
+</lu-page-header>
+			`,
+		};
+	},
+} as Meta;
+
+export const Basic = {
+	args: {
+		label: 'H1. Page title',
+		labelPortalContent: false,
```

Comment:
It's quite confusing to have 2 controls for 1 input imho. 
I'd rather go for a single one, but adding global documentation for `portalContent`.